modules/network-api/src/main/java/org/apache/ignite/network/MessagingService.java

```diff
@@ -65,6 +65,20 @@ public interface MessagingService {
      */
     CompletableFuture<Void> respond(ClusterNode recipient, NetworkMessage msg, long correlationId);
 
+    /**
+     * Sends a response to a {@link #invoke} request.
+     * Guarantees are the same as for the {@link #send(ClusterNode, NetworkMessage)}.
+     *
+     * <p>If the recipient cannot be resolved (because it has already left the physical topology), the returned future is resolved
+     * with the corresponding exception.
```

Comment:
Can you please clarify the type of the exception? It's not clear right now, what should you handle